.github/workflows/main.yml

```diff
@@ -7,58 +7,63 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        php: [7.3, 7.4, 8.0, 8.1]
+        php: [7.3, 7.4, 8.0, 8.1, 8.2]
         stability: [prefer-lowest, prefer-stable]
     name: Tests on PHP ${{ matrix.php }} - ${{ matrix.stability }}
     steps:
-      - name: Checkout
-        uses: actions/checkout@v2
+      - uses: actions/checkout@v2
```

Comment:
These are pretty far out of date, will the v4 upgrade we needed cause any issues here or was that only for the repos that deal with tags?

Reply:
We need it, but Dependabot will come in post-merge of this and handle all of that. I was losing track what needed v3 and what needed v4 so was going to let it handle it.